Xamarin.Forms.Core/MultiPage.cs

```diff
@@ -159,15 +161,17 @@ protected virtual void UnhookContent(T content)
 		{
 		}
 
-		internal static int GetIndex(T page)
+		[EditorBrowsable(EditorBrowsableState.Never)]
+		public static int GetIndex(T page)
```

Comment:
Do you have a preference for how we expose static members? We could just leave them as is for now and if a customer needs those methods just tell them to use the isense hidden ones.